File: TestApplication/app/src/main/java/me/henryfbp/testapplication/MainActivity.java
package me.henryfbp.testapplication;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;
import android.widget.LinearLayout;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    Button button;
    LinearLayout linearLayout;

    public void appendThing(View v) {

        TextView text = new TextView(this);
        text.setText("I am an even testier string!");

        text.setLayoutParams(new LinearLayout.LayoutParams(
                LinearLayout.LayoutParams.MATCH_PARENT,
                LinearLayout.LayoutParams.MATCH_PARENT
        ));

        linearLayout.addView(text);

    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        button = findViewById(R.id.button);
        linearLayout = findViewById(R.id.linearLayout);

    }
}

File: labs/final/Final/app/src/main/java/me/henryfbp/final_app/MainActivity.kt
package me.henryfbp.final_app

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View.INVISIBLE
import android.view.View.VISIBLE
import android.widget.Button
import android.widget.ProgressBar

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val buttonStart = findViewById<Button>(R.id.buttonStart)
        val buttonStop = findViewById<Button>(R.id.buttonStop)
        val progressBar = findViewById<ProgressBar>(R.id.progressBar)

        buttonStart.setOnClickListener {

            progressBar.visibility = VISIBLE

            progressBar.progress = 10
        }

        buttonStop.setOnClickListener {
            progressBar.progress = 0

            progressBar.visibility = INVISIBLE
        }
    }
}
